pinot-core/src/main/java/com/linkedin/pinot/core/io/readerwriter/impl/FixedByteSingleColumnSingleValueReaderWriter.java

```diff
@@ -29,191 +29,109 @@
 import org.slf4j.LoggerFactory;
 
 
-// TODO Check if MMAP allows us to pack more partitions on a single machine.
+/**
+ * This class implements reader as well as writer interfaces for fixed-byte single column and single value data.
+ * <ul>
+ *   <li> Auto expands memory allocation on-demand. </li>
+ *   <li> Supports random reads and writes. </li>
+ *   <li> Rows not written but allocated default to 0. </li>
```

Comment:
not sure if it guaranteed to be 0, maybe dependent on the mem manager. may want to make that clear that it is user's responsibility to use it appropriately.